src/raft.rs

```diff
@@ -2107,19 +2096,21 @@ impl<T: Storage> Raft<T> {
     ///
     /// ```rust
     /// use raft::{Raft, Config, storage::MemStorage, eraftpb::ConfState};
-    /// let config = Config {
+    /// use raft::raw_node::new_mem_raw_node;
```

Comment:
We shouldn't show users how to use our library with testing only functions.

Please revert this.